bin/protoc

```diff
@@ -3,7 +3,7 @@
 require 'platform'
 require 'protoc/version'
 
-PLATFORM_TO_PROTOC_ARCH = {:x86 => :x86_32, :x86_64 => :x86_64}
+PLATFORM_TO_PROTOC_ARCH = {:x86 => :x86_32, :x86_64 => :x86_64, :powerpc => :powerpc}
```

Comment:
I think you need to make similar changes in "ext/protoc/extconf.rb" to prevent us from trying to build the binary even though we already have it.